action.yml

```diff
@@ -19,12 +19,16 @@ inputs:
     required: false
     description: 'If the action should skip PRs that have failed checks, defaults to `true`'
     default: 'true'
+  require_auto_merge_enabled:
+    required: false
+    description: 'If the action should skip PRs that have failed checks, defaults to `true`'
```

Comment:
Was this description copied and pasted from `require_passed_checks`? Please update it accordingly.

Reply:
@nickqi-tubi thanks for the catch. Copilot did that, I thought it was correct

Reply:
Updated, thanks